packages/backend-common/src/reading/UrlReaders.ts

```diff
@@ -24,6 +24,7 @@ import { GithubUrlReader } from './GithubUrlReader';
 import { GitlabUrlReader } from './GitlabUrlReader';
 import { ReadTreeResponseFactory } from './tree';
 import { FetchUrlReader } from './FetchUrlReader';
+import { GcsUrlReader } from './GcsUrlReader';
```

Comment:
Nit: perhaps we should rename this to `googleGcsUrlReader` ? :)

In TechDocs we've been using this "Provider + Service" pattern as well - `googleGcs`, `awsS3`, `azureBlobStorage`, etc.